esmvaltool/recipes/recipe_Ocean_scarlar_fields.yml

```diff
@@ -0,0 +1,60 @@
+###############################################################################
+# recipe_scarlar_fields.yml
+---
+documentation:
+  description: |
+    Recipe to demonstrate several simple plots based on the scalar fields.
+    These fluxes are saved as MFO files in CMIP5.
+    Please use this file as a template for adding additional
+    fields into the ocean. This work based on the BGC-val toolkit GMD-2018-103.
+    Written by Lee de Mora, Plymouth Marine Laboratory
+
+  authors:
+    - ledm
+
+  references:
+    - BGC-val:gmd-2018-103, https://doi.org/10.5194/gmd-11-4215-2018, 2018.
+
+  projects:
+    - ukesm
+
+
+datasets:
+  # working datasets
+    - {dataset: ACCESS1-0,        project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: ACCESS1-3,        project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: CanESM2,          project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: GFDL-ESM2M,       project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: HadGEM2-CC,       project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: HadGEM2-ES,       project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: MPI-ESM-LR,       project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: MPI-ESM-MR,       project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: MPI-ESM-P,        project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: MRI-CGCM3,        project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: MRI-ESM1,         project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: NorESM1-M,        project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+    - {dataset: NorESM1-ME,       project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1900, end_year: 2004}
+
+
+preprocessors:
+  # --------------------------------------------------
+  # Time series preprocessors
+  # --------------------------------------------------
+  prep_timeseries: # Load file as is.
+    custom_order: true
```

Comment:
There is no need to define a preprocessor if you use the default values

Reply:
Removed!